cmd/mimir/config-descriptor.json

```
@@ -5412,6 +5412,28 @@
               "fieldFlag": "blocks-storage.tsdb.max-tsdb-opening-concurrency-on-startup",
               "fieldType": "int",
               "fieldCategory": "advanced"
+            },
+            {
+              "kind": "field",
+              "name": "out_of_order_cap_min",
+              "required": false,
+              "desc": "Minimum capacity for out of order chunks (in samples. between 0 and 255.)",
```

```suggestion
              "desc": "Minimum capacity for out-of-order chunks, in samples between 0 and 255.",
```
Please vet for accuracy. The punctuation in the original is problematic.